doc/other_modules/other_modules.md

```diff
@@ -1,7 +1,15 @@
 # Other Modules
 
+- [flame_oxygen](flame_oxygen/flame_oxygen.md) - Want to utilize the Flame
+  Component System in a different project?  This package will do that!
+- [flame_splash_screen](flame_splash_screen/flame_splash_screen.md) - Add the
+  beautiful "Powered by Flame" splash screen.
+- [Tiled](tiled.md)
+
 ```{toctree}
-Oxygen         <oxygen.md>
-Tiled          <tiled.md>
-Splash screen  <splash_screen.md>
+:hidden:
+
+flame_oxygen        <flame_oxygen/flame_oxygen.md>
```

Comment:
It's not flame_oxygen that should be linked here, but the stand-alone Oxygen package.

Reply:
Ok, then this needs a bridge package entry as well then.

Reply:
> Edit: I think splash screen should be moved to bridge libraries because it does technically bridge to a Flutter widget outside of the flame engine, right?

Yup, it just hasn't been updated and moved in yet.

Reply:
Ok, fixing that then.